Poker.BE/Poker.BE.Domain/Game/Pot.cs

```diff
@@ -8,11 +8,46 @@ namespace Poker.BE.Domain.Game
 {
     public class Pot
     {
+        #region Fields
+        private Pot partialPot;
+        private Pot basePot;
+        private int value;
+        private int amountToClaim;
+        private List<Player> playersClaimPot;
+        #endregion
+
         #region Properties
-        public Pot PartialPot { get; set; }
+        public Pot PartialPot { get { return this.partialPot; } set { this.partialPot = value; } }
```

Comment:
this is not apply when you want to have a special `get { return ... } set { ... = value}`. there you need to have a private field as the base in-memory variant for this property. [as explain here](http://stackoverflow.com/questions/367192/why-does-property-set-throw-stackoverflow-exception).